libraries/AP_RangeFinder/AP_RangeFinder_JRE_Serial.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "AP_RangeFinder.h"
+#include "AP_RangeFinder_Backend_Serial.h"
+
+#ifndef AP_RANGEFINDER_JRE_SERIAL_ENABLED
```

Comment:
It's great to see this driver!

I think we should move this _ENABLED definition to AP_RangeFinder_config.h.  The pattern has changed just in the last two weeks but still.. it would be good to stick with the new pattern.

Reply:
I moved this _ENABLED definition to AP_RangeFinder_config.h.